cli/pkg/kctrl/cmd/package/init.go

```diff
@@ -215,6 +224,22 @@ func (o *InitOptions) newOrExistingPackageResources() (*v1alpha1.Package,
 	return pkg, pkgMetadata, pkgInstall, nil
 }
 
+func (o *InitOptions) updateAndDefaultPackageResources(pkgRefName string, pkgMetadata *v1alpha1.PackageMetadata, pkg *v1alpha1.Package) {
+	pkgMetadata.Name = pkgRefName
+	pkg.Spec.RefName = pkgRefName
+	pkgMetadata.Spec.DisplayName = strings.Split(pkgRefName, ".")[0]
+
+	shortDesc := pkgMetadata.Spec.ShortDescription
+	if len(shortDesc) == 0 {
```

Comment:
inline? 
What if this was removed by the user?

Reply:
Good question, but I cannot imagine how we will figure that out 🤔 
Since we are only setting doing this for values that are displayed by kctrl while listing/getting

I guess nudging users towards giving them values is not the worst idea 😆

Reply:
I guess we can discuss this and put up an issue if we wanna give it more thought

Reply:
Made this inline